lib/fog/kubevirt/compute/compute.rb

```diff
@@ -401,7 +401,7 @@ def watch_vms(opts = {})
         #
         def watch_vminstances(opts = {})
           mapper = Proc.new do |notice|
-            vminstance = OpenStruct.new(Vminstance.parse(notice.object)) if notice.object.kind == 'VirtualMachineInstance'
+            vminstance = OpenStruct.new(Vminstance.parse(object_to_hash(notice.object))) if notice.object.kind == 'VirtualMachineInstance'
```

Comment:
I wonder why this change was required only now. Does it mean that other watched entities needs this as well ? 
If this is the case, there might be an issue with miq-providers-kubevirt provider that uses them.

Reply:
Parsing fails. Possibly we introduced a regression some time ago. Other watches could be broken as well but this should be done in different PR. I only fixed bugs to make the tests pass.

Reply:
It seems like we introduced regression in vm_parse. We need to fix `watch_vms` as well.